dat/main.py

```diff
@@ -119,6 +124,7 @@ def create_external_tables(external_tables_path, output_path):
 cli.add_command(write_generated_reference_tables)
 cli.add_command(write_schemas)
 cli.add_command(create_external_tables)
+cli.add_command(write_generated_reference_table_3)
```

Comment:
IMO we should rethink the "generate a single reference table" later, and just have this reference table be part of the tables generated by the current command, which generates all of them.